src/styles/page-template/index.md.njk

```diff
@@ -36,161 +36,138 @@ You can customise the page template, for example, by adding a service name and n
 
 {{ example({group: "styles", item: "page-template", example: "custom", customCode: true, html: true, nunjucks: true, open: false, size: "xl" }) }}
 
-## Nunjucks
+## Changing template content
 
+If you’re [using Nunjucks](https://frontend.design-system.service.gov.uk/use-nunjucks-macros/), you can change the template’s content using options.
 
-### Configuring Nunjucks
+You can use either:
 
-When using Nunjucks you’ll need to add `node_modules/govuk-frontend/`
-as available views:
+- `set` to use a single value or string
+- `block` to use a multiline value or HTML markup
 
-```js
-nunjucks.configure([
-  "node_modules/govuk-frontend/"
-], {
-  autoescape: true
-})
-```
-
-See the [Nunjucks getting started guide](https://mozilla.github.io/nunjucks/getting-started.html) for more information.
-
-Once you have configured Nunjucks, you can now extend `template.njk` in your view:
+For example, to use `set` to add a string before the closing </body> element in the template:
 
-```
+```javscript
 {% raw %}
-{% extends "govuk/template.njk" %}
+{% set bodyEnd = "Example text" %}
 {% endraw %}
 ```
 
-### Changing template content
-
-The Nunjucks template allows 2 ways to change the content: [variables](https://mozilla.github.io/nunjucks/templating.html#set) or [blocks](https://mozilla.github.io/nunjucks/templating.html#block).
+To use `block` to add a block of HTML before the closing </body> element in the template:
 
-The main difference between a variable and a block is that blocks can contain markup.
-
-#### Variables
+```javscript
+{% raw %}
+{% block bodyEnd %}
+  <div>
+     <p>Example text</p>
+  </div>
+{% endblock %}
+{% endraw %}
+```
 
-Variables can be set with:
+To change the components that are included in the page template by default, set their equivalent blocks. For example:
 
-```js
+```javscript
 {% raw %}
-{% set variableName = "value" %}
+{% block header %}
+  {{ header({
+    homepageUrl: "/custom-url"
+  }) }}
+{% endblock %}
 {% endraw %}
 ```
 
+### Options
+
 <table class="govuk-table app-table--constrained">
-  <caption class="govuk-table__caption govuk-visually-hidden">Variables that can be used with Template</caption>
+  <caption class="govuk-table__caption govuk-visually-hidden">Options that you can use with the page template</caption>
```

Comment:
Some of these are blocks and some of them are variables. I think unfortunately that means we need to keep this as two separate tables and use intentionally different language to describe them so the user understands that they're configured using two different syntaxes.

Reply:
Makes sense! So we can keep the combined alphabetical table (which I think will help users find the option they're looking for, because they don't need to kinda 'guesstimate' whether an option is a variable or a block first), I've added an 'option type' column. Hope that works!